src/main.js

```diff
@@ -0,0 +1,119 @@
+//  after getting to extension to update rows and discs added the wincondition property and used Object assign to make a copy of the original board so that my board property could be manipulated with moveDisc without affecting the winCondition property
+
+var Game = function () {
+  this.board = [[5, 4, 3, 2, 1], [], []];
+  this.originalBoardState = [[5, 4, 3, 2, 1], [], []];
+  this.displayBoard = function () {
+    var display = this.board.map(function (row) {
+      return "---" + " " + row.join(" ");
+    });
+    display.forEach(function (row) {
+      console.log(row);
+    });
+  };
+  this.moveDisc = function (currentPeg, newPeg) {
+    if (
+      typeof currentPeg === "number" &&
+      typeof newPeg === "number" &&
+      currentPeg <= this.board.length &&
+      currentPeg > 0 &&
+      currentPeg !== newPeg &&
+      newPeg <= this.board.length &&
+      newPeg > 0
+    ) {
+      var currentPegPlacement = this.board[currentPeg - 1];
+      var topDisc = this.board[currentPeg - 1].length - 1;
+      var newPegPlacement = this.board[newPeg - 1];
+      var newTopDisc = this.board[newPeg - 1].length - 1;
+      var discToMove = currentPegPlacement[topDisc];
+      var topOfNewPeg = newPegPlacement[newTopDisc];
+
+      if (currentPegPlacement.length === 0) {
+        console.log("you can not make a disc appear out of thin air");
+      }
+      if (topOfNewPeg === undefined || discToMove < topOfNewPeg) {
+        //store the current pegs top disc in a var
+        // console.log(discToMove);
+        var savedDisc = currentPegPlacement.pop();
+        newPegPlacement.push(savedDisc);
+        // var discBeingMoved = discToMove.pop();
+        // topOfNewPeg.push(discBeingMoved);
+        console.log("That Move was successful, board is now:");
+      } else {
+        console.log(
+          "You cannot move a larger disc on top of a smaller one, board is still:"
+        );
+      }
+      this.displayBoard();
+      this.checkWinner();
+    } else {
+      console.log("please only use the available pegs");
+    }
+  };
+  //did not have time to add constraints to this  like the movedisc function ..no error handling
+  this.changePlayStyle = function (pegs, discs) {
+    var boardArray = [];
+    for (let i = 1; i <= pegs; i++) {
+      const peg = [];
```

Comment:
stay consistent with you variable declaration, you switch between var and const (pre es6 vs es6).